functional/FunctionalTest.java

```diff
@@ -2984,6 +3050,7 @@ public static void runTests() throws Exception {
     copyObject_test8();
     copyObject_test9();
 
+
```

Comment:
We can remove the extra empty lines in middle

Reply:
done.